godot-core/src/builtin/glam_helpers.rs

```diff
@@ -55,7 +55,8 @@ pub(crate) trait GlamType {
 }
 
 macro_rules! impl_glam_map_self {
-    ($T:ty) => {
+    ($(,)?) => {};
+    ($($T:ty),+ $(,)?) => {$(
```

Comment:
What's the reason behind this? I'd rather repeat the macro invocation than making the macro more complex.

Repetition patterns are mostly useful for token ranges _inside_ the macro body, not for repeating the entire macro.

Reply:
i can change it if you want, but it's not that uncommon afaik. the standard library does it a fair bit, see for instance https://doc.rust-lang.org/src/core/cmp.rs.html#1370.